lib/react/renderer.rb

```diff
@@ -22,9 +23,9 @@ def self.setup!(react_js, components_js, args={})
       @@pool = ConnectionPool.new(default_pool_options.merge(args)) { self.new }
     end
 
-    def self.render(component, args={})
+    def self.render(component, args={}, replay_console=false)
```

Comment:
Thanks for feedback. I wanted to keep it out of `args` because that's meant for use by the React component directly. The other place that seems reasonable would be on the app.config.react object, then passing into `Renderer.setup!`